index.js

```diff
@@ -138,14 +151,14 @@ function addDependencies (loader, paths, callback) {
           )
         }
         remaining--
-        if (remaining === 0) callback(null)
+        if (remaining === 0) { callback(null) }
       }
     })
   })
 }
 
 var setTimeoutMsFromTimeoutInPlace = util.deprecate(function (config) {
-  if (config.timeoutMs != null) {
+  if (config.timeoutMs) {
```

Comment:
I believe this change is in error because if fails to flag the following ambiguous config:

```js
options: {
  timeout: 5,
  timeoutMs: 0
}
```